e2store/src/era2.rs

```diff
@@ -164,22 +173,27 @@ impl TryFrom<&Entry> for AccountEntry {
     }
 }
 
-impl TryInto<Entry> for AccountEntry {
+impl TryFrom<AccountEntry> for Entry {
     type Error = anyhow::Error;
 
-    fn try_into(self) -> anyhow::Result<Entry> {
-        let rlp_encoded = alloy_rlp::encode(self);
-        let buf: Vec<u8> = vec![];
-        let mut encoder = snap::write::FrameEncoder::new(buf);
+    fn try_from(value: AccountEntry) -> Result<Self, Self::Error> {
+        let rlp_encoded = alloy_rlp::encode(value);
+        let mut encoder = snap::write::FrameEncoder::new(vec![]);
         let _ = encoder.write(&rlp_encoded)?;
         let encoded = encoder.into_inner()?;
         Ok(Entry::new(0x08, encoded))
     }
 }
 
 #[derive(Clone, Eq, PartialEq, Debug)]
-pub struct StorageEntry {
-    pub storage: Vec<StorageItem>,
+pub struct StorageEntry(Vec<StorageItem>);
```

Comment:
Now that I think about it, you will have to do one of those two:
- Make it `pub struct StorageEntry(**pub** Vec<StorageItem>);`
- Create constructor: `pub fn new(items: Vec<StorageItem>) -> Self {`

Otherwise, other files won't be able to create instances of it (which they will need).
But you can also add it once you have a need for it.

Reply:
I will do this in the PR that uses this code then in that case